themes/wporg-translate-events-2024/blocks/event-title/index.php

```diff
@@ -19,6 +19,12 @@
 			}
 			$url = Urls::event_details( $event->id() );
 			?>
+			<?php
+			if ( 'remote' === $event->attendance_mode() ) {
+				// phpcs:ignore WordPress.Security.EscapeOutput.OutputNotEscaped
+				echo do_blocks( sprintf( '<!-- wp:wporg-translate-events-2024/remote-attendance-icon %s  /-->', wp_json_encode( array( 'inline_css' => 'float:left; margin-right:3px' ) ) ) );
```

Comment:
I think we should add some margin in the top. You can see the 5px in the row (1) vs the current margin in the row (2). 

![image](https://github.com/user-attachments/assets/aa981c40-2084-4965-bcf3-0bd3eed99757)

```suggestion
				echo do_blocks( sprintf( '<!-- wp:wporg-translate-events-2024/remote-attendance-icon %s  /-->', wp_json_encode( array( 'inline_css' => 'float:left; margin:5px 3px 0 0' ) ) ) );
```

One question about adding the inline CSS: what do you thing about extracting these CSS to a CSS file instead of adding them inline?

Reply:
Resolved in #377